tests/e2e/General/UsageTest.php

```diff
@@ -328,15 +328,16 @@ public function testStorageStats(array $data): array
         $this->assertEquals(1, count($response['body']['requests']));
         $this->assertEquals($requestsTotal, $response['body']['requests'][array_key_last($response['body']['requests'])]['value']);
         $this->validateDates($response['body']['requests']);
-        $this->assertEquals($storageTotal, $response['body']['filesStorageTotal']);
+        // CANNOT check as now we use sizeActual which is not available in response
+        // $this->assertEquals($storageTotal, $response['body']['filesStorageTotal']);
```

Comment:
Can we maybe check that it's increased?

Reply:
I think it should decrease, with compression?

Reply:
I meant that if `storageTotal` before upload is N, then after upload, `storageTotal` should be bigger than N.